.github/workflows/e2e.yml

```diff
@@ -94,6 +106,17 @@ jobs:
           command: install
           args: --git https://github.com/cosmos/sp1-ics07-tendermint --rev ${{ env.SP1_OPERATOR_REV }} sp1-ics07-tendermint-operator --bin operator --locked
 
+      - name: Setup Kurtosis
+        if: env.ETH_TESTNET_TYPE == 'pos'
+        run: |
+          echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
+          sudo apt update
+          sudo apt install kurtosis-cli=1.3.0 # Must match the kurtosis library version we use in the e2e tests
+          kurtosis engine start
+          kurtosis analytics disable
+          echo "$(dirname $(which kurtosis))" >> $GITHUB_PATH
+        shell: bash
+
```

Comment:
There isn't an action to setup kurtosis?

Reply:
No, the only one I found also runs a lot of tests and other things we don't want